src/main/java/uk/gov/hmcts/reform/sscs/models/serialize/ccd/Address.java

```diff
@@ -13,4 +17,17 @@
     private String town;
     private String county;
     private String postcode;
+
+    @JsonCreator
+    public Address(@JsonProperty("line1") String line1,
+                   @JsonProperty("line2") String line2,
+                   @JsonProperty("town") String town,
+                   @JsonProperty("county") String county,
+                   @JsonProperty("postcode") String postcode) {
```

Comment:
What is the purpose of all the JsonProperty?

Reply:
Could you clarify @sathishbabuj why you need those jsonProperties, please?

Reply:
To transform the CaseDetails objects from CCD to CaseData  object easily.

Reply:
could you point to the line where that's is need, please?

Reply:
is it used here -> https://github.com/hmcts/sscs-case-loader/blob/sscs-2934-multiple-hearings/src/main/java/uk/gov/hmcts/reform/sscs/services/ccd/CcdCasesSender.java#L56 ??

Reply:
I think @sathishbabuj added those json properteis because they are needed now to deserialize the caseDetails into CaseData as done in the link I provided above. @sathishbabuj is that correct?

Reply:
Yes @iguannago . To be precise https://github.com/hmcts/sscs-case-loader/blob/8293736afa1934fa070b1a8ab7fdfa8ff21c44d5/src/main/java/uk/gov/hmcts/reform/sscs/services/ccd/CcdCasesSender.java#L58